.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce
+
+*Please use markdown syntax, especially for Backtraces and such!*
+
+### program versions
+
+    - operating system
+    - neomutt
```

Comment:
We need to be a bit careful.
Most people will have the binary `mutt` not `neomutt`.
Most people will have an older version which doesn't have `-D -S`.